pkg/appmgmt/general/general.go

```diff
@@ -146,6 +146,8 @@ func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata,
 				zap.String("namespace", pod.Namespace),
 				zap.String("name", pod.Name),
 				zap.Error(err))
+			events.GetRecorder().Eventf(pod, nil, v1.EventTypeWarning, "TaskGroupFormatError", "TaskGroupFormatError",
+				"unable to get taskGroups for pod, pod namespace: %s, pod name: %s, reason: %s", appID, pod.Namespace, pod.Name, err.Error())
```

Comment:
I think we have a discrepancy here between the number of args passed in and the formatting directives in the string:
```
unable to get taskGroups for pod, pod namespace: <appID>, pod name: <pod.Namespace>, reason: pod.Name
```
I think there is a `%s` missing between "pod" and the first comma

Reply:
Sorry I made a mistake. Modified. Thanks.